text/0000-roadmap-2021.md

```diff
@@ -0,0 +1,127 @@
+
+### Where does this group work?
+
+This can include the meeting platforms used and chat channels. It’s the contact information for the whole group, not just the contact person noted previously.
+
+### What does this group do?
+
+The goal is to provide a summary of scope for the team. In particular, this is the section that other teams and contributors will read to make decisions for when to consult this team.
+
+This section may also include links to subgroups where that makes sense, for example the compiler and dev-tools teams have a large number of subgroups which should be documented here.
+
+### How does this group make decisions?
+
+This section establishes the process (or processes) by which the team makes decisions. This is useful for both team members and outside parties. Any normative documents produced by the team (for example, the Reference for lang team) should also be listed here.
+
+# Unifying processes and vocabulary
+
+Today, Rust teams decide individually where to host documents and have discussions. We do not want to take away these choices from the Rust teams: we believe that teams know best what works for them. However, sometimes the divergence here is unintentional, and providing defaults that work well can be an excellent way to remove friction.
+
+The specific details of the vocabulary and processes that we will seek to unify are not yet clear, primarily because there is no one place where all of the specifics of team activity are laid out. As we begin to establish charters over the course of the year, we will begin to identify common needs and current choices, and will work to diverge only when needed.
+
+# Unified project tracking
+
+A frequent concern -- even amongst our most active team members -- is that knowing what is going on in teams can be difficult. It can be difficult to identify what efforts are active or temporarily paused, and what kind of feedback is sought for on proposals.
+
+As part of our charter and unified process/vocabulary work, we would like to work to surface the ongoing work in teams in a common way. The aim is to target contributors/maintainers, both existing and new, not people aiming to propose a new project. (It may be a useful gauge of how much bandwidth the team has left, though).
+
+However, it is expected that the road towards tracking team efforts for all of our teams is going to require more work than we can fit in this coming year. However, a number of our teams have already established or are working to establish similar tracking dashboards: the language, library, and crates.io teams. (TODO: links). We expect to unify the structure of these existing dashboards and publish guiding documentation which other teams can use as a framework for making their own; by the end of the year we want to add a compiler team dashboard but do not expect to be able to expand the set further.
```

Comment:
I'd be interested in having this for rustdoc; right now a lot of my plans are just in my head and will probably never get implemented if I leave the team. I guess the rust-lang/rust issue tracker works, but there's so many issues that things get lost really easily.

Reply:
My hope is that these will be on repositories like rust-lang/rustdoc-team or so, so you can have project boards; I need to add in the links to the RFC itself, but you can see a few examples:

* https://github.com/rust-lang/lang-team/projects/2
* https://github.com/rust-lang/crates.io/projects/2
* https://github.com/rust-lang/libs-team/projects/2

Reply:
Oh nice!